src/geometries/CapsuleGeometry.js

```diff
@@ -0,0 +1,35 @@
+import { Path } from '../extras/core/Path.js';
+import { LatheGeometry } from './LatheGeometry.js';
+
+class CapsuleGeometry extends LatheGeometry {
+
+	constructor( radius = 1, length = 1, capSegments = 4, radialSegments = 8 ) {
+
+		const path = new Path();
+		path.absarc( 0, 0, radius, Math.PI * 1.5, 0 );
+		path.absarc( 0, length, radius, 0, Math.PI * 0.5 );
+
+		super( path.getPoints( capSegments ), radialSegments );
+
+		this.translate( 0, - length / 2, 0 );
```

Comment:
Is there a way of doing this offset in the `Path` instead? 🤔

Reply:
https://github.com/mrdoob/three.js/pull/23586/commits/4ed33917f351a3c47477a5e70497df1fd93fd9e6 LGTY?

Reply:
LGTM!